v3.1/Dockerfile

```diff
@@ -51,8 +53,9 @@ COPY modsecurity.d /etc/apache2/modsecurity.d
 COPY custom-rules/before-crs.dist /opt/modsecurity/rules/before-crs.dist
 COPY custom-rules/after-crs.dist /opt/modsecurity/rules/after-crs.dist
 COPY transform-alert-message.awk /opt/
+COPY virus-check.pl /opt
```

Comment:
This can be added to the previous `COPY` directive:
```
COPY transform-alert-message.awk virus-check.pl /opt/
```

Reply:
combined the two COPY directives.